chat-plugins/mafia-data.js

```diff
@@ -1,208 +1,112 @@
+		],
+		image: '<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-mafia.png"/>',
 	},
-	foolWin: function () {
-		this.game.end('<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-fool.png" />', "The fool has been lynched and is victorious!");
+	werewolf: {
+		name: 'Werewolf',
+		plural: 'Werewolves',
+		id: 'werewolf',
+		color: '#FFA500',
+		memo: [
+			'Factional Communication: If there are other Werewolf-aligned players, you may PM them during the game.',
+			'Factional Kill: The Werewolves may kill one player per Night.',
+			'You are aligned with the <span style="color:$FFA500;font-weight:bold">Werewolves</span>. You win when all players without a Werewolf wincon are eliminated and at least one Werewolf-aligned player is still alive (or nothing can prevent the same).',
+		],
+		image: '<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-werewolf.png"/>',
+	},
+	alien: {
+		name: 'Alien',
+		plural: 'Aliens',
+		id: 'alien',
+		color: '#F0F',
+		memo: [
+			'Factional Communication: If there are other Alien-aligned players, you may PM them during the game.',
+			'Factional Kill: The Aliens may kill one player <span style="text-decoration:underline">once during the game</span>.',
+			'You are aligned with the <span style="color:#F0F;font-weight:bold">Aliens</span>. You win when all players without a Alien wincon are eliminated and at least one Alien-aligned player is still alive (or nothing can prevent the same).',
+		],
+		// TODO image for alien faction
+		image: '<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>',
+	},
+	cult: {
+		name: 'Cult',
+		plural: 'Cultists',
+		id: 'cult',
+		color: '#000',
+		memo: [
+			// TODO cult is a bit tricky and rare... confirm details of abilities before adding more memo (if any).
+			'You are aligned with the <span style="color:#000;font-weight:bold">Cult</span>. You win when all players without a Cult wincon are eliminated and at least one Cult-aligned player is still alive (or nothing can prevent the same).',
+		],
+		// TODO image for cult faction
+		image: '<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>',
+	},
+	solo: {
+		// Special alignment for all roles that are on their own.
+		name: 'Solo',
+		plural: 'Solos',
+		id: 'solo',
+		image: '<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-goon.png"/>',
 	},
 };
 
-// Every role has a side they belong to, as well as all functions they have. These functions are objects with the targeting mechanics and a callback.
-// events are atStart, onNight, onDay, onLynch.
-exports.MafiaClasses = {
-	__proto__: null,
-
+exports.roles = {
+	vt: 'villager',
+	villy: 'villager',
 	villager: {
```

Comment:
Vanilla Townie should also be used as an alias of Villager (as it is sometimes used)

Reply:
While I do agree, there is an issue. The role parser splits the role by its spaces, so that becomes `["Vanilla", "Townie"]`, I can add townie as an alias though, and look into setting vanilla as a modifier (iirc there is a vanilla modifier anyways).